srcpkgs/j/template

```diff
@@ -40,9 +40,9 @@ do_configure() {
 	     /jbuilder/s@unknown@${_jbuilder}@" \
 	    jsrc/jversion-x.h > jsrc/jversion.h
 
-	sed -i "/^install=./s@/usr/share/j/[0-9.]\+@${_jshare}@
-	        /^user=./s@home,userx@home,'/${_juser}'@" \
-	       jlibrary/bin/profile.ijs
+	sed "/^install=./s@/usr/share/j/[0-9.]\+@${_jshare}@
+	     /^user=./s@home,userx@home,'/${_juser}'@" \
+	    jlibrary/bin/profile.ijs > jlibrary/bin/profile.ijs.new
```

Comment:
These `sed` calls should use maybe `vsed` ?

Reply:
xbps-src needs bash because POSIX shell does not have some features we use.

Reply:
This is one of them.

Reply:
Ha. The looks like vsed normalizes newlines to spaces, so we need semicolons now. An unexpected gotcha, but done.